src/stdune/id.mli

```diff
@@ -0,0 +1,23 @@
+open !Set
+
+module type S = sig
+  type t
+
+  module Set : Set_intf.S with type elt = t
```

Comment:
```suggestion
  module Set : Set.S with type elt = t
```

(`X_intf` modules are for internal purpose only, they are just to avoid duplicating signatures. In general they shouldn't be used outside of `X`)

Reply:
Changed.